avaframe/ana4Stats/probAnaCfg.ini

```diff
@@ -20,24 +20,40 @@ peakLim = 1.0
 ##relTh = 0.75|0.9|0.8
 
 
-[PROBRUN]
-# settings for probabilistic run of computational module - add here parameters for parameter variation
-# on parameter is varied at a time - other parameters are kept at reference values
-# list of parameters that shall be varied - separated by |
-varParList = mu|relTh
-#++++++VARIATION RANGE
-# provide range used for variation - will be used +- these values from
-# reference value (which is read from computational module settings)
-# in PERCENT percentVariation = True, if False - variation using absolute values
-percentVariation = True
-muVariation = 35
-relThVariation = 35
-#++++++NUMBER OF STEPS
-# how many steps in variation - if reference value not included will be appended
-muSteps = 11
-relThSteps = 11
-# if reference values shall be added to variation set addStandardConfig to True
-addStandardConfig = False
+# [PROBRUN]
+# # IF USED - UNCOMMENT THIS FULL SECTION
+# # settings for probabilistic run of computational module - add here parameters for parameter variation
+# # one parameter is varied at a time - other parameters are kept at reference values
+# # list of parameters that shall be varied - separated by |
+# varParList = mu|relTh
+# #++++++VARIATION INFO
+# variationType options: percent, range, normaldistribution
+# # if percent: +- varParVariation percent of varParValue in varParSteps
+# # if range: +- varParVariation of varParValue in varParSteps
+# # if normaldistribution: vary varParValue in varParSteps following a normal dist defined by
+# # varParValue and varParVariation (corresponding to buildType in NORMAL DIST parameters below)
+# # varParValue is the reference value (which is read from computational module settings)
+# variationType = percent
+# muVariation = 35
+# relThVariation = 35
+# #++++++NUMBER OF STEPS
+# # how many steps in variation - if reference value not included will be appended
+# muSteps = 11
+# relThSteps = 11
+# #if reference values shall be added to variation set addStandardConfig to True
+# addStandardConfig = False
```

Comment:
Does this need to be commented?

Reply:
the problem is if it is not commented it is not possible to add other variation and step values for parameters that are not in the default settings, if the full Section is 'new', additional keys are allowed

Reply:
I agree here: I don't think a completely commented out section is a good idea, this should be solved differently then.

Reply:
ok now solved differently varParList, variationValue and numberOfSteps each separated by | for each parameter